datasets/pings.md

```diff
@@ -10,3 +10,48 @@
 
 You can find the reference documentation for all ping types
 [here](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/concepts/pings.html).
+
+# Ping Metadata
+
+The telemetry data pipeline appends a `meta` key to the ping body containing
+information about the ingestion environment, including timestamps;
+Geo-IP data about the client;
+and fields extracted from the ping or client headers that are useful for downstream processing.
+
+Since the `meta` key is not present in the ping as it is sent by the client,
+these fields are documented here.
+
+As of September 28, 2018, members of the `meta` key on main pings include:
+
+
+Key | Description
+--- | -----------
+`appBuildId` |
+`appName` | e.g. "Firefox"
+`appUpdateChannel` |
+`appVendor` | e.g. "Mozilla"
+`appVersion` |
+`clientId` |
+`creationTimestamp` | Client `creationDate` field, transformed to nanoseconds since epoch
+`Date` | Client HTTP header reflecting the client time when the ping is sent, like `Fri, 28 Sep 2018 14:01:57 GMT`
+`DNT` | Not present in all pings
+`docType` | e.g. "main"
+`documentId` | A UUID identifying the ping, generated by the client
+`geoCity` | from Geo-IP lookup of client IP; `??` if unknown
+`geoCountry` | from Geo-IP lookup of client IP; `??` if unknown
+`geoSubdivision1` | from Geo-IP lookup of client IP; not present in all pings
+`geoSubdivision2` | from Geo-IP lookup of client IP; not present in all pings
+`Host` | Public hostname of the ingestion endpoint
+`Hostname` | Private hostname of the ingestion endpoint
+`normalizedChannel` |
+`normalizedOSVersion` |
+`os` |
+`reason` | Documented [in the source tree](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/data/main-ping.html)
+`sampleId` | `crc32(clientId) % 100`
+`sourceName` | e.g. "telemetry"
+`sourceVersion` | Client `version` field (reflecting the version of the ping format)
+`submissionDate` | Server date (GMT) when ping was received, like `20180928`
+`telemetryEnabled` | Extracted value of `environment.settings.telemetryEnabled`, describing whether opt-in telemetry is enabled
+`Timestamp` | Server timestamp when the ping is received, expressed as nanoseconds since epoch
+`Type` | e.g. "telemetry"
+`X-PingSender-Version` | Not present in all pings
```

Comment:
Maybe link to more info about what `PingSender` is? Or just to [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1350044)?